src/System.Private.ServiceModel/src/project.json

```diff
@@ -53,7 +53,8 @@
       "dependencies": {
         "System.Net.NameResolution": "4.0.0-rc3-23809",
         "System.Net.Security": "4.0.0-rc3-23809",
-        "System.Net.Sockets": "4.1.0-rc3-23809"
+        "System.Net.Sockets": "4.1.0-rc3-23809",
+        "System.Security.Principal.Windows": "4.0.0-rc3-23809"
```

Comment:
Dumb question, but is S.S.Principal.Windows now supported on *nix?  A few weeks ago this dependency broke WCF and a few other CoreFx libraries.